pkgs/development/libraries/openssl/default.nix

```diff
@@ -122,8 +123,21 @@ let
       "MANSUFFIX=ssl"
     ];
 
+    buildFlags = lib.optionals static [
+      # Even though engines are disabled in static builds, we have to
+      # override ENGINESDIR so the bin output doesn't end up with an
+      # reference to the lib output.
+      "ENGINESDIR=/"
+    ];
+
     enableParallelBuilding = true;
 
+    preInstall = ''
```

Comment:
Should this be also conditional on `static`?

Reply:
Good catch, thanks.